conda/cli/main_env_update.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+"""CLI implementation for `conda-env update`.
+
+Updates the conda environments with the specified packages.
+"""
+import os
+import sys
+import textwrap
+from argparse import RawDescriptionHelpFormatter
+
+from conda.base.context import context, determine_target_prefix
+from conda.cli.conda_argparse import (
+    add_parser_json,
+    add_parser_prefix,
+    add_parser_solver,
+)
+from conda.core.prefix_data import PrefixData
+from conda.exceptions import CondaEnvException
+from conda.misc import touch_nonadmin
+from conda.notices import notices
+
+sys.path.append("..")
+from conda.env import specs as install_specs
+
+from ..env.installers.base import InvalidInstaller, get_installer
+from .common import get_filename, print_result
+
+description = """
+Update the current environment based on environment file
+"""
+
+example = """
+examples:
+    conda env update
+    conda env update -n=foo
+    conda env update -f=/path/to/environment.yml
+    conda env update --name=foo --file=environment.yml
+    conda env update vader/deathstar
+"""
+
+
+def configure_parser(sub_parsers):
+    p = sub_parsers.add_parser(
+        "update",
+        formatter_class=RawDescriptionHelpFormatter,
```

Comment:
The `RawDescriptionHelpFormatter` is already the default formatter_class, so this is no longer necessary:

https://github.com/conda/conda/blob/abd5873b70f4663b1cfb179086b366424780a2fa/conda/cli/conda_argparse.py#L206-L211